howto_draw_image.md

```diff
@@ -0,0 +1,8 @@
+# How to draw an image
+
+With Aspose.Imaging you can use several methods to draw graphics primitives such as `DrawLine`, `DrawEllipse`, `DrawRectangle`, `DrawArc`, `DrawBezier` and `DrawString`. You need to create a graphic surface for drawing and set up graphic properties. We will create a surface of 100 x 100 pixels with 32 bits per pixel in our example and set the background to yellow color with `Clear` method. Then we can define drawing a tool, for example, a Pen with a Blue color, to draw a dotted line or use SolidBrush for continuous lines with different colors, followed by X,Y coordinates of the start and end points of the lines:
+
+{% gist aspose-com-gists/ebaa56878c36388b3fcab68c7a5adb66 drawing-lines.cs %}
+
+For more C# code examples please follow the link:
+[drawing-images](https://docs.aspose.com/imaging/net/drawing-images/)
```

Comment:
@anms23 , for seo purposes does not links should be done with corresponding text? I mean for example links like link - The more detailed description of resize, crop and rotate operations you can find in the official documentation link etc?

Reply:
@sam199008 , I've changed documentation link formatting:
https://github.com/aspose-imaging/aspose-imaging-probe/commit/077113a66de28454f13ea2b934d7f8bc5dbdd5c5